src/Api/WebClient.php

```diff
@@ -220,41 +197,28 @@ private function createShipment()
         return $shipment;
     }
 
-    /**
-     * @return string
-     */
-    private function generateGuid()
+    private function generateGuid(): string
```

Comment:
Can you extract it into new class and spec it?

Reply:
According to our later conversation, we leave it as it is

Reply:
Done